### ISSUE TITLE
android:allowBackup in library causes conflict with app's Manifest

### ISSUE THREAD

**Issue by skromberg** [palaima/DebugDrawer#52]
[More details ](https://code.google.com/p/android/issues/detail?id=70073)

```
<application
        tools:replace="android:label,android:allowBackup"
        android:name=".Application"
        android:allowBackup="false"
        android:icon="@mipmap/ic_launcher"
        android:label="@string/app_name">
```

and

```
Error:Execution failed for task ':app:processDevDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:23:9-36 is also present at [io.palaima.debugdrawer:debugdrawer:0.7.0] AndroidManifest.xml:11:18-44 value=(true). Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:20:5-65:19 to override.
```

**Comment by palaima** [comment]
Fix: 46c64682b60c0cfafd39646040734afd99be06ab

**Comment by slavkoder** [comment]
Update: nevermind, forgot to upgrade library version 😅 

@palaima the problem still exists. The fix you provided does not resolve the issue, since the default value for `android:allowBackup` is `"true"` ([reference](https://developer.android.com/guide/topics/manifest/application-element#allowbackup))
